src/main/java/org/nguh/nguhcraft/block/NguhBlocks.kt

```diff
@@ -860,8 +903,8 @@ object NguhBlocks {
         it.addAll(CHAINS_AND_LANTERNS.flatten())
         it.addAll(ALL_BROCADE_BLOCKS)
 
-        // Slabs may drop 2 or 1 and are thus handled separately.
-        it.addAll(ALL_VARIANT_FAMILY_BLOCKS.filter { it !is SlabBlock })
+        // Slabs may drop 2 or 1 and are thus handled separately. Same for doors
+        it.addAll(ALL_VARIANT_FAMILY_BLOCKS.filter { !(it is SlabBlock || it is DoorBlock) })
```

Comment:
```suggestion
        it.addAll(ALL_VARIANT_FAMILY_BLOCKS.filter { it !is SlabBlock && it !is DoorBlock })
```
Negating parenthesised conditions like this always gives me a headache.